source/Halibut.Tests/Support/LatestClientAndLatestServiceBuilder.cs

```diff
@@ -601,6 +642,7 @@ public class ClientAndService : IClientAndService
                 ForceClientProxyType? forceClientProxyType,
                 CancellationTokenSource cancellationTokenSource)
             {
+                PollingClient = pollingClient;
```

Comment:
Is there a way we can pass enough generic detail to achieve the same result? It doesn't feel quite right to start passing polling specific objects into this more generic class

Reply:
do you really just need `clientUri`

Reply:
Indeed. Done!